jgrapht-ext/src/test/java/org/jgrapht/ext/DOTImporterTest.java

```diff
@@ -156,9 +156,6 @@ public void updateVertex(
        importer.read(input, result);
 
        Assert.assertEquals(expected.toString(), result.toString());
-
```

Comment:
Hm, I was thinking of: Assert.assertEquals(expected, result);
assertEquals will invoke expected.equals(result), where the equals method is implemented in AbstractBaseGraph